src/components/Buttons.tsx

```diff
@@ -0,0 +1,61 @@
+import { ButtonHTMLAttributes, ReactNode, forwardRef } from "react";
+
+type ButtonProps = ButtonHTMLAttributes<HTMLButtonElement> & {
+  children: ReactNode;
+};
+
+export const PrimaryButton = forwardRef<HTMLButtonElement, ButtonProps>(
+  ({ children, ...props }, ref) => (
```

Comment:
Consider combining `PrimaryButton` and `PrimaryButtonWide` into a single component with a `wide` boolean prop.